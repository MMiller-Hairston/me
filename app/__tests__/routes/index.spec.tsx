import renderer from 'react-test-renderer';
import Index from '~/routes';

test('Index should render', () => {
    const component = renderer.create(<Index />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});
