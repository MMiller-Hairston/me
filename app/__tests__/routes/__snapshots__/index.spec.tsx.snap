// Vitest Snapshot v1

exports[`Index should render 1`] = `
<main
  className="max-w-7xl mx-auto"
  id="main"
>
  <section
    className="h-screen"
  >
    <div
      className="grid justify-center gap-4 items-center pt-40 px-4"
    >
      <h1
        className="text-3xl font-bold"
      >
        Michael Miller-Hairston
      </h1>
      <p>
        Father / Software Engineer / Tinkerer
      </p>
      <p
        className="font-semibold"
      >
        Arizona based Software Engineer
         
        <a
          className="underline"
          href="https://www2.deloitte.com/us/en.html"
        >
          @Deloitte
        </a>
        .
      </p>
      <div
        className="flex justify-center gap-6"
      >
        <a
          href="mailto:michael.millerhairston@gmail.com"
        >
          <span
            title="Email"
          >
            <svg
              aria-hidden="true"
              className="h-6 w-6"
              fill="none"
              stroke="currentColor"
              strokeWidth={1.5}
              viewBox="0 0 24 24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M21.75 6.75v10.5a2.25 2.25 0 01-2.25 2.25h-15a2.25 2.25 0 01-2.25-2.25V6.75m19.5 0A2.25 2.25 0 0019.5 4.5h-15a2.25 2.25 0 00-2.25 2.25m19.5 0v.243a2.25 2.25 0 01-1.07 1.916l-7.5 4.615a2.25 2.25 0 01-2.36 0L3.32 8.91a2.25 2.25 0 01-1.07-1.916V6.75"
                strokeLinecap="round"
                strokeLinejoin="round"
              />
            </svg>
          </span>
        </a>
      </div>
    </div>
  </section>
</main>
`;
